frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/TestResultInputGroup.tsx

```diff
@@ -0,0 +1,145 @@
+import { Alert } from "@trussworks/react-uswds";
+
+import {
+  MultiplexResultInput,
+  SupportedDiseaseTestPerformed,
+} from "../../../../generated/graphql";
+import { DevicesMap } from "../../QueueItem";
+import RadioGroup, {
+  RadioGroupOptions,
+} from "../../../commonComponents/RadioGroup";
+import { TEST_RESULTS } from "../../../testResults/constants";
+import { TEST_RESULT_DESCRIPTIONS } from "../../../constants";
+
+interface Props {
+  testOrderId: string;
+  testResults: MultiplexResultInput[];
+  deviceId: string;
+  devicesMap: DevicesMap;
+  onChange: (value: MultiplexResultInput[]) => void;
+}
+
+type TestResultRecord = Record<string, TEST_RESULTS>;
+
+const convertTestResultRecordToArray = (testResultRecord: TestResultRecord) => {
+  return Object.entries(testResultRecord).map(
+    ([diseaseName, testResult]): MultiplexResultInput => {
+      return {
+        diseaseName: diseaseName,
+        testResult: testResult,
+      };
+    }
+  );
+};
+
+const convertMultiplexResultArrayToRecord = (
+  testResults: MultiplexResultInput[]
+) => {
+  const testResultRecords: TestResultRecord = {};
+  testResults.forEach((value) => {
+    if (value.diseaseName) {
+      testResultRecords[value.diseaseName] =
+        (value.testResult as TEST_RESULTS) ?? TEST_RESULTS.UNKNOWN;
+    }
+  });
+  return testResultRecords;
+};
+
+const filterDuplicateDiseaseTests = (
+  diseaseTests: SupportedDiseaseTestPerformed[]
+) => {
+  const uniqueDiseases = new Set();
+  const filteredList = [];
+
+  for (const diseaseTest of diseaseTests) {
+    if (!uniqueDiseases.has(diseaseTest.supportedDisease.name)) {
+      uniqueDiseases.add(diseaseTest.supportedDisease.name);
+      filteredList.push(diseaseTest);
+    }
+  }
+
+  return filteredList;
+};
+
+export const TestResultInputGroup = ({
+  testOrderId,
+  testResults,
+  deviceId,
+  devicesMap,
+  onChange,
+}: Props) => {
+  const unfilteredDiseaseTests =
+    devicesMap.get(deviceId)?.supportedDiseaseTestPerformed ?? [];
```

Comment:
we aren't filtering out rsv and hiv based on the feature flag so they are showing up as result options in the test card (like how we are filtering them out in the device options)

Reply:
Great find on this! I just pushed a fix and will redeploy on dev2